packages/loader/container-loader/src/createAndLoadContainerUtils.ts

```diff
@@ -239,3 +279,133 @@ export async function loadFrozenContainerFromPendingState(
 		documentServiceFactory: new FrozenDocumentServiceFactory(),
 	});
 }
+
+/**
+ * Loads a summarizer container with the required headers, triggers an on-demand summary, and then closes it.
+ * Returns success/failure and an optional error for host-side handling.
+ *
+ * @legacy @beta
```

Comment:
Is it necessary to put it in beta directly? ShouldCam this be experimental initially? @anthony-murphy.